spynnaker/pyNN/models/recording_common.py

```diff
@@ -61,6 +67,10 @@ def _record(self, variable, new_ids, sampling_interval, to_file):
         # Get bit array of indices to record for this variable
         indices = self._indices_to_record[variable]
 
+        # update sampling interval
+        if sampling_interval is not None:
+            self._sampling_interval = sampling_interval
```

Comment:
Probably don't need to do it again...

Reply:
Actually Worse. Not sure sampling_interval belongs here at all.

Reply:
Will look at this in the selective recording PR

Reply:
In Selective record sampling intervals are held in NeuronRecorder and no longer in RecordingCommon